src/plugins/subsystems/drm/drm.sh

```diff
@@ -44,6 +51,22 @@ function drm_main()
 
   [[ -n "${options_values['VERBOSE']}" ]] && flag='VERBOSE'
 
+  if [[ -n "$decode_edid" ]]; then
+    #module_control 'LOAD' "$target" "$remote" "$load_module" "$flag"
+    decode_edid
+    if [[ "$?" != 0 ]]; then
+      return 22
```

Comment:
Please, add `# EINVAL` to all the returns comments (if you use something different from 22, use the correct label).